README.md

```diff
@@ -41,9 +41,7 @@ resolver
 ```typescript
 import { parsers, resolvers, ResolverEngine } from "@resolver-engine/core";
 
-const resolver = new ResolverEngine<string>()
-  .addResolver(resolvers.UriResolver())
-  .addParser(parsers.UrlParser());
+const resolver = new ResolverEngine<string>().addResolver(resolvers.UriResolver()).addParser(parsers.UrlParser());
```

Comment:
seems less readable?

Reply:
I second that. Imho it looked better before.

Reply:
I concurr, however this is what Prettier insists on having